packages/babel-parser/test/fixtures/experimental/destructuring-private/invalid-async-call-arguments/output.json

```diff
@@ -0,0 +1,106 @@
+{
+  "type": "File",
+  "start":0,"end":46,"loc":{"start":{"line":1,"column":0},"end":{"line":4,"column":1}},
+  "errors": [
+    "SyntaxError: Private names can only be used as the name of a class element (i.e. class C { #p = 42; #m() {} } )\n or a property of member expression (i.e. this.#p). (3:16)"
```

Comment:
When the plugin is enabled, we should update the error message to also mention destructuring.

Reply:
The current error message does not cover `#p in o`, either. At this point I feel like listing all the valid cases in error messages is a bit verbose. Should we just say unexpected private field?

Reply:
Uh yes, if adding every case it's too much lets stay vague to cover all of them.